src/site/content/en/_redirects.yaml

```diff
@@ -173,3 +173,8 @@ redirects:
 
 - from: /discover-installable/
   to: /progressive-web-apps/#installable
+
+# Redirect for spec name changes
+
+- from: /sms-receiver-api-announcement
+  to: /web-otp
```

Comment:
I think they're all supposed to have `/` at the end, @robdodson - can you confirm?

I know for sure the `to: ...` should have a forward slash at the end.

Reply:
I _think_ either will work. @samthor to confirm or deny :)

Reply:
Most of the redirects in the file *lack* a trailing `/`.